packages/manager/apps/dedicated/client/app/dedicated/server/details/server.service.js

```diff
@@ -261,8 +261,19 @@ export default class ServerF {
       data: {
         monitoring,
       },
-      broadcast: 'dedicated.informations.reload',
-    });
+    }).then(() =>
+      this.OvhHttp.get('/dedicated/server/{serviceName}', {
+        rootPath: 'apiv6',
+        urlParams: {
+          serviceName,
+        },
+      }).then((data) => {
+        return this.$rootScope.$broadcast(
```

Comment:
Can't we reload the state or somethign instead of broadcasting through `$rootScope` ? Or at least move this callback to the call so we can  maybe use  `$scope.$emit` instead

Reply:
i understand this point but with actual tructure i have not find solution to do this differently with my actual capability :'(

Reply:
if is a blocker i can cancel this PR and this fix became a subject for a bigger rework :/

Reply:
We don't need to call the API again or refresh the route. I made the same fix in PR https://github.com/ovh/manager/pull/4835. Please take a look at it.